extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParser.java

```diff
@@ -172,7 +173,9 @@ public OpenCensusProtobufInputRowParser withParseSpec(ParseSpec parseSpec)
       // One row per timeSeries point.
       for (Point point : ts.getPointsList()) {
         // Time in millis
-        labels.put(TIMESTAMP_COLUMN, point.getTimestamp().getSeconds() * 1000);
+        final Timestamp t = point.getTimestamp();
+        final long millis = Instant.ofEpochSecond(t.getSeconds(), t.getNanos()).toEpochMilli();
```

Comment:
Do we usually declare locals as `final`?  I know it's accurate, but typically not part of coding convention.  :man_shrugging:

Reply:
In this case it's not a requirement, I think it's just my IDE default when extracting variables. It does matter when using references inside of anonymous classes.

Reply:
Right.  And can also be useful documentation when the variable is initialized separate from declaration.